### PR TITLE
[v0.0.29-ms1] Disabling scanning and vibration features.

### PR DESCRIPTION
**Pull request by maxsdev**
I add opportunity for disable vibration in props.
Also i add opportunity for enabling and disabling scanning because I collided with he problem when qr code scanning, then i show allert and qr code scanning again and app all the time vibrate and update alert. But I need, that scanning will continue when I dismiss alert.

### CODE REVIEW

**Comment by eXhausted** [comment]
Thanks, @maxsdev 👍

@moaazsidat merge it plz

File: index.js [hunk 1]
```diff
@@ -92,6 +94,7 @@ export default class QRCodeScanner extends Component {
       fadeInOpacity: new Animated.Value(0),
       isAuthorized: false,
       isAuthorizationChecked: false,
+      disablingByUser: false,
```
Comment (moaazsidat) [inline index.js]: can we give this a more descriptive/explicit name like `disableVibrationByUser`, thanks!

Comment (eXhausted) [reply index.js]: @maxsdev would you please prepare change? :)

**Comment by CoderXpert** [comment]
@maxsdev Would be great if you add how to use this feature, will be easier for newbies to pick this up.

**Comment by moaazsidat** [comment]
since this PR has been inactive, if other users of the library want this feature, I'll accept a new PR making the changes + the requested changes.

**Pull request closed without merging**